add generator for nsm site notes & generate notes (#258)
networkservicemesh/site: content/docs/releases/v1.11.0.md

+++
short="v1.11.0"
title = "Release v1.11.0"
date="2023-10-06 00:00:00 +0000 UTC"
+++

# NSM Release v1.11.0 

NSM v1.11.0 has been tested on:
- [kind](https://github.com/networkservicemesh/integration-k8s-kind/actions?query=branch%3Arelease%2Fv1.11.0+)
- [GKE](https://github.com/networkservicemesh/integration-k8s-gke/actions?query=branch%3Arelease%2Fv1.11.0+)
- [AKS](https://github.com/networkservicemesh/integration-k8s-aks/actions?query=branch%3Arelease%2Fv1.11.0+)
- [AWS/AWS IPv6](https://github.com/networkservicemesh/integration-k8s-aws/actions?query=branch%3Arelease%2Fv1.11.0+)
- [Bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/actions?query=branch%3Arelease%2Fv1.11.0+)
- [Interdomain GKE/AWS/AKS](https://github.com/networkservicemesh/integration-interdomain-k8s/actions?query=branch%3Arelease%2Fv1.11.0+)

## Changes since last release

### Add LoadBalancer for vl3 networks

NSM vl3 could be perceived as an alternative to k8s networks. Currently, k8s service is the most useful thing in the k8s network.

> At this moment, NSM vl3 netowk doesn't provide any alternative to k8s services. So we need to consider and implement the best solution for it.

Reviewer: ```suggestion
At this moment, NSM vl3 network doesn't provide any alternative to k8s services. So we need to consider and implement the best solution for it.
```

Author: Fixed, thanks.

[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9210)

### Add interdomain healing examples

Currently, we don't test healing over interdomain or floating interdomain. So we could add some examples of how to do that.

Scenarios:
- Forwarders death in floating interdomain scenario
- NSE death in floating interdomain scenario
- NSM systems death in floating interdomain scenario
- Proxy nsmgrs death in interdomain scenario
- NSMGRs death in interdomain scenario
- Registry death in interdomain scenario

> We need to check scenarious from single cluster testing in multicluster scenarious.

Reviewer: ```suggestion
We need to check scenarios from single cluster testing in multi cluster scenarios.
```

Author: Fixed, thanks.

[See more details](https://github.com/networkservicemesh/sdk-vpp/issues/737)

## System stability fixes and improvements

### No artifacts after CI run on public clusters

Status: RESOLVED.

[See more details](https://github.com/networkservicemesh/integration-tests/issues/2874)

### `Update dependent repositories` workflow does update incorrectly

`integration-k8s-kind` re-uses [.github workflow](https://github.com/networkservicemesh/.github/blob/main/.github/workflows/update-dependent-repositories-gomod.yaml)
This is wrong, because this workflow updates `integration-k8s-kind` instead of `integration-tests` in dependent repositories (public clusters).
For example:
https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/5656999510/job/15325140836

[See more details](https://github.com/networkservicemesh/integration-k8s-kind/issues/862)

###  How can i make composition of endpoints?

Status: RESOLVED.

[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9676)

### IPSec takes a lot of time on request

> Our current implementation uses IKEv2, that requires `rsa` key. 

Reviewer: ```suggestion
Our current implementation uses IKEv2 which requires an `rsa` key. 
```

Author: Fixed, thanks.
